vita3k/kernel/src/load_self.cpp

```diff
@@ -583,12 +583,12 @@ SceUID load_self(KernelState &kernel, MemState &mem, const void *self, const std
             dump_segments[seg_index].p_vaddr = segment.addr;
             last_index = std::max(seg_index, last_index);
         }
-        constexpr auto DUMP_DIR = "elfdumps";
+        auto DUMP_DIR = fs::path(dump_path) / "elfdumps";
```

Comment:
DUMP_DIR can be lowercased. it's not a constant anymore.

Reply:
Done.